lib/widgets/HomeScreen/home_screen.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:classroom_app/widgets/CustomButton/custom_button.dart';
+import 'package:flutter/material.dart';
+
+class HomeScreen extends StatelessWidget {
```

Comment:
Use `MediaQuery` to make your layout adaptable to different screen sizes

Reply:
done